activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -293,6 +293,11 @@ def primary_key(table_name)
       def create_table(table_name, id: :primary_key, primary_key: nil, force: nil, **options, &block)
         validate_create_table_options!(options)
         validate_table_length!(table_name) unless options[:_uses_legacy_table_name]
+
+        if force && options.key?(:if_not_exists)
+          raise ArgumentError, "Only one of `:force` or `:if_not_exists` options can be provided, but not both."
```

Comment:
[non-blocking] Not to bikeshed on this, but I found the error message slightly confusing. My take on this:

```suggestion
          raise ArgumentError, "Options `:force` and `:if_not_exists` cannot be used simultaneously."
```

Reply:
Thanks, looks nicer!